doc/architecture/decisions/0028-secure-pipelines-and-control-tower.md

```diff
@@ -0,0 +1,49 @@
+# 28. Secure pipelines and Control tower
+
+[Next >>](9999-end.md)
+
+Date: 2023-06-18
+
+## Status
+
+Proposed
+
+## Context
+
+The existing platform is trying to align with the rest of DI, which uses AWS Control Tower to manage AWS accounts, and
+Secure Pipelines to deploy their systems. As a result, the platform is attempting to align to this flow, while
+also aligning our architecture (as laid out in [ADR 2#](Put link here)). DI also uses an environment per account flow,
+which is different from our approach of an account for all our non-prod environments, and an account for our prod
+environment.
```

Comment:
If we decide we aren't going to migrate the existing thing, then this ADR should probably reflect that @oskarwilliams . I still think Control Tower, then pipelines, then rewriting code into Lambdas/TypeScript is the correct order, but either way let's just make sure the ADRs reflect the decision we're actually making :)

Reply:
So we are going to migrate the existing terraform into secure pipelines, independent of the order we do things (as a lot of this will be parallel), so I will leave this as is